lib/matplotlib/figure.py

```diff
@@ -935,40 +939,40 @@ def get_facecolor(self):
         return self.patch.get_facecolor()
 
     def get_figwidth(self):
-        """Return the figwidth as a float."""
+        """Return the figure width as a float."""
         return self.bbox_inches.width
 
     def get_figheight(self):
-        """Return the figheight as a float."""
+        """Return the figure height as a float."""
         return self.bbox_inches.height
 
     def get_dpi(self):
-        """Return the dpi as a float."""
+        """Return the resolution in dots per inch as a float."""
         return self.dpi
 
     def get_frameon(self):
-        """Get the boolean indicating frameon."""
+        """Return a bool indicating whether the figure frame will be dawn."""
```

Comment:
I tend to write these just as "return whether ..." which seems less verbose and just as clear.
Also typo: dawn.